families/track_and_trade/client/src/views/list_agents.js

```diff
@@ -17,6 +17,7 @@
 'use strict'
 
 const m = require('mithril')
+const _ = require('lodash')
```

Comment:
Using the explicit require will allow the minimization via webpack to produce a much smaller file.